src/pages/Detailed/header/DetailedHeader.js

```diff
@@ -0,0 +1,158 @@
+      <div className={styles.wrapper}>
+        <div className={styles.left}>
+          <DIV className={styles.logo}>
+            <ProjectIcon name={name || ''} ticker={ticker} size={40} />
+            <div className={styles.name}>
+              <h1>{name}</h1>
+              <DIV className={styles.description}>{description}</DIV>
+            </div>
+          </DIV>
+
+          <div className={styles.price}>
+            <div className={styles.priceUsd}>
+              {priceUsd && formatNumber(priceUsd, { currency: 'USD' })}
+              <PercentChanges
+                className={styles.percentChanges}
+                changes={percentChange24h}
+              />
+              <Label className={styles.label} accent='waterloo'>
+                24h
+              </Label>
+            </div>
+            {!loading && project && (
+              <div className={styles.percentsBottom}>
+                <Label className={styles.supply}>
+                  {formatNumber(totalSupply)} {ticker}
+                </Label>
+                <PercentChanges
+                  className={styles.percentChangesSmall}
+                  changes={percentChange7d}
+                />
+                <Label className={styles.label} accent='waterloo'>
+                  7d
+                </Label>
+              </div>
+            )}
+          </div>
+        </div>
+        {isLoggedIn && isDesktop && !loading && (
+          <div className={styles.right}>
+            <ChartSignalCreationDialog
+              slug={slug}
+              trigger={
+                <Button border accent='ghost' className={styles.signalButton}>
+                  <img
+                    className={styles.icon}
+                    src={addSignalSvg}
+                    alt='Add signal'
+                  />
+                  Add signal
+                </Button>
+              }
+            />
+
+            <WatchlistsPopup
+              trigger={
+                <Button
+                  accent='positive'
+                  variant='fill'
+                  className={styles.watchlistButton}
+                >
+                  <img
+                    className={cx(styles.icon, styles.watchlistIcon)}
+                    src={addWatchlistSvg}
+                    alt='Watch slug'
+                  />
+                  Watch {name}
+                </Button>
+              }
+              projectId={id}
+              slug={slug}
+              isLoggedIn={isLoggedIn}
+            />
+          </div>
+        )}
+      </div>
+    </>
+  )
+}
+
+export default compose(
+  createSkeletonProvider(
+    {
+      project: {
+        name: '',
+        description: '______ ___ ______ __ _____ __ ______',
+        ticker: '',
+        percentChange24h: 0,
+        percentChange7d: 0,
+        priceBtc: 0,
+        priceUsd: 0
+      }
+    },
+    ({ loading }) => loading,
+    () => ({
+      backgroundColor: '#bdc3c7',
+      color: '#bdc3c7'
+    })
```

Comment:
that skeleton not so nice, when page is loading.
![image](https://user-images.githubusercontent.com/24521041/63329548-dd4f6c00-c33a-11e9-8a86-061357e2a0b5.png)
 Maybe temporary we will add `PageLoader`/ `Loader` from `san-ui` for some elements? and in future will make new skeleton

Reply:
Will change to PageLoader. Thank you.